dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -3819,12 +3831,21 @@ RtpsUdpDataLink::RtpsWriter::gather_directed_heartbeat_i(const SingleSendBuffer:
                                                          const ReaderInfo_rch& reader)
 {
   const SequenceNumber first_sn = reader->durable_ ? 1 : non_durable_first_sn(proxy);
-  const SequenceNumber last_sn = expected_max_sn(reader);
+  SequenceNumber last_sn = expected_max_sn(reader);
```

Comment:
`ACE_UNUSED_ARG` can help avoid a warning when security and asserts are both disabled

Reply:
Nevermind... I missed that it was used on another line